lib/onlyoffice_webdriver_wrapper/webdriver.rb

```diff
@@ -750,12 +750,17 @@ def webdriver_error(exception, error_message = nil)
     end
 
     def wait_until(timeout = ::PageObject.default_page_wait, message = nil, &block)
+      tries ||= 3
```

Comment:
n the context of exception handling, def method_name ... end is functionally equivalent to begin ... end. Both can include rescue statements for example.

http://stackoverflow.com/questions/34628349/what-is-use-of-begin-end-block-in-ruby